src/app/components/settings/classes/classes.component.html

```diff
@@ -36,15 +36,16 @@ <h3>Set Classes</h3>
 				<input class="form-control" placeholder="Last Name" name="teacherLastName" [value]="class.teacher.lastName" [(ngModel)]="class.teacher.lastName">
 			</td>
 			<td class="class-manage">
-				<button class="btn btn-info selected"
-					[class.active]="class._id === aliasClass?._id"
-					(click)="aliasClass = class; aliasModal.show()"
-					[disabled]="savingClasses || !class._id"
-					tooltip="Save the class before linking it!"
-					[isDisabled]="savingClasses || !class._id">
-					<i class="fa fa-link"></i>
-					Link
-				</button>
+				<div style="display: inline-block" >
```

Comment:
This could be entirely appropriate, but I was just wondering what the function was for this `div` with `display: inline-block;`? If it is important for layout or something, I'd suggest you move the CSS to its respective file and give the `div` a class name.

Reply:
I think it was temporary, but it's been so long I forget

Reply:
It might have been something with the tooltips, but test them without the div just in case

Reply:
it's a container for the tooltip, and removing that caues all sorts of formatting issues.